src/builder/struct.ts

```diff
@@ -156,6 +162,16 @@ export class Struct implements IStructBuilder, HasProperties {
     return this.add(updatedProp);
   }
 
+  /**
+   * Update all existing properties
+   */
+  public updateAll(update: Partial<Property>) {
```

Comment:
Should we exclude `name` here?
```suggestion
  public updateAll(update: Partial<Omit<Property, 'name'>>) {
```